README.md

```diff
@@ -1 +1,162 @@
 # ApiTestCase
+[![Build Status](https://travis-ci.org/Lakion/ApiTestCase.svg?branch=master)](https://travis-ci.org/Lakion/ApiTestCase)
+
+**ApiTestCase** is an one-file library that will make your life as a Symfony2 API developer much easier. It extends basic [Symfony2](https://symfony.com/) WebTestCase with some cool features. Thanks to [*PHP-Matcher*](https://github.com/coduo/php-matcher) you can write expected json responses like a gangster(According to php-matcher README.md). [*SymfonyMockerContainer*](https://github.com/PolishSymfonyCommunity/SymfonyMockerContainer) makes it super easy to mock services. This library merges these two great concepts and extends them with a few more lines, to achieve pure awesomeness.
+
+Installation
+------------
+
+Assuming you already have Composer installed globally:
+
+```bash
+$ composer require lakion/api-test-case
+```
+
+Then you have to slightly change your Kernel logic to support SymfonyMockerContainer:
+
+```php
+// app/AppKernel.php
+
+protected function getContainerBaseClass()
+{
+    if ('test' === $this->environment) {
+        return '\PSS\SymfonyMockerContainer\DependencyInjection\MockerContainer';
+    }
+
+    return parent::getContainerBaseClass();
+}
+```
+And its done! ApiTestCase is working with the default configuration.
+
+Configuration reference
```

Comment:
I'd also move it to the bottom, after installation I want to see usage if it works out of the box.